application/apps/indexer/definitions/src/record.rs

```diff
@@ -79,22 +79,17 @@ impl ParseOperationResult {
 ///
 /// If different output is required (e.g., exporting DLT data as raw bytes), a different
 /// parser implementation (such as `DltRaw`) should be used.
+#[derive(Debug)]
 pub enum LogRecordOutput<'a> {
     /// A raw binary message.
     Raw(&'a [u8]),
 
-    /// A borrowed UTF-8 string slice.
-    Str(&'a str),
-
-    /// A string wrapped in a [`Cow`] (Copy-on-Write), allowing either borrowed or owned data.
-    Cow(Cow<'a, str>),
-
-    /// An owned UTF-8 string.
-    String(String),
+    /// Log message line
+    Message(Cow<'a, str>),
 
     /// Structured columnar data. Typically used when the parser can extract
-    /// meaningful fields and present them as an array of strings.
-    Columns(&'a [&'a str]),
+    /// meaningful fields and present them as collection of texts.
+    Columns(Vec<Cow<'a, String>>),
```

Comment:
What is the reason? Plugins?
Btw, we also should remove "columns" logic from DLT & SomeIp parsers. Yes, now in prototype they are suing `String`, but they should return `Columns` instead, and it might be `Columns(&'a [&'a str])` or `Columns(&'a [Cow<'a>])` - we don't know yet. In any way, this is a part of internal API and it my opinion it makes sense to keep it not limited, but quite "open". It costs zero, but gives a good semantic overview of the supported parser's outputs.